static/js/github-edit-button.js

```diff
@@ -0,0 +1,31 @@
+(function addGithubEditButton() {
+    $('#edit-on-github').click(function (e) {
+        if (location.hostname.indexOf("localhost") != -1 ) 
+        {
+            alert("Disabled for localhost")
+            return;
+        }
+        var path = location.pathname;
+        markdownFile = path.replace(/\/$/, '.md');        
+        if (!path.endsWith(".html")) path += "/index.html";
```

Comment:
This line now has no side effect on the program,  because path var is no longer used